kubeadm/flannel/flannel-host-gw.yml

```diff
@@ -0,0 +1,171 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-windows-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+data:
+  run.ps1: |
+    $ErrorActionPreference = "Stop";
+
+    mkdir -force /host/etc/cni/net.d
+    mkdir -force /host/etc/kube-flannel
+    mkdir -force /host/opt/cni/bin
+    mkdir -force /host/flannel
+    mkdir -force /host/flannel/var/run/secrets/kubernetes.io/serviceaccount
+
+    $cniJson = get-content /etc/kube-flannel-windows/cni-conf.json | ConvertFrom-Json
+    $serviceSubnet = yq r /etc/kubeadm-config/ClusterConfiguration networking.serviceSubnet
+    $podSubnet = yq r /etc/kubeadm-config/ClusterConfiguration networking.podSubnet
+    $networkJson = wins cli net get | convertfrom-json
+
+    $cniJson.delegate.policies[0].Value.ExceptionList = $serviceSubnet, $podSubnet, $networkJson.SubnetCIDR
+    $cniJson.delegate.policies[1].Value.DestinationPrefix = $serviceSubnet
+    $cniJson.delegate.policies[2].Value.DestinationPrefix = $networkJson.AddressCIDR
+    Set-Content -Path /host/etc/cni/net.d/10-flannel.conf ($cniJson | ConvertTo-Json -depth 100)
+
+    cp -force /etc/kube-flannel/net-conf.json /host/etc/kube-flannel
+    cp -force -recurse /cni/* /host/opt/cni/bin
+    cp -force /flannel/flanneld.exe /host/flannel/flanneld.exe
+    cp -force /kube-proxy/kubeconfig.conf /host/flannel/kubeconfig.yml
+    cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/flannel/var/run/secrets/kubernetes.io/serviceaccount/
+    wins cli process run --path C:\flannel\flanneld.exe --args "--kube-subnet-mgr --kubeconfig-file /flannel/kubeconfig.yml" --envs "POD_NAME=$env:POD_NAME POD_NAMESPACE=$env:POD_NAMESPACE"
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "cniVersion": "0.3.0",
```

Comment:
the `cniVersion` in `kubeadm/flannel/Dockerfile` is 0.8.5.
can we have 0.8.5 here as well?

Reply:
That's the version of the [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md), 0.8.5 is the version of the [CNI plugins bundle](https://github.com/containernetworking/plugins/releases/tag/v0.8.5)